app/services/more_like_this_getter.rb

```diff
@@ -0,0 +1,87 @@
+# MoreLikeThisGetter.new(work).works
+#
+# returns an array of zero to ten
```

Comment:
Why ten? What determines that it's 10?  The actual Solr config? Something else?

We don't actually want 10 more-like-this works displayed. What part of the code is limiting to only 3 or X?  I am having trouble finding it. 

Would it make sense for hte MoreLikeThisGetter to actually take an argument to one of it;s methods asking for how many you want at most?  (If limiting to only, say, 3, it should be the 3 that Solr rates as _most_ alike. Do we have a way to make sure that's so, test it?)

Reply:
I did add a "maximum items" argument to the instantiation method for the getter object.

Reply:
Oh, it's not obvious where the default value of 10 retrieved items is and how it might be possible to ask for fewer results to be returned. (No, you can't use a `mlt.count` parameter).